clients/java/signalr/src/test/java/com/microsoft/signalr/HubConnectionTest.java

```diff
@@ -913,8 +913,8 @@ public void errorWhenReceivingInvokeWithIncorrectArgumentLength()  {
 
         hubConnection.start().timeout(1, TimeUnit.SECONDS).blockingAwait();
 
-        RuntimeException exception = assertThrows(RuntimeException.class, () -> mockTransport.receiveMessage("{\"type\":1,\"target\":\"Send\",\"arguments\":[]}" + RECORD_SEPARATOR));
-        assertEquals("Invocation provides 0 argument(s) but target expects 1.", exception.getMessage());
+        mockTransport.receiveMessage("{\"type\":1,\"target\":\"Send\",\"arguments\":[]}" + RECORD_SEPARATOR);
```

Comment:
Filed https://github.com/aspnet/SignalR/issues/3225

Reply:
Is this your version of :shipit:
:trollface: